### PR TITLE
text → z͚̳ͤ̓͐a̳̺ͥl̞̪͗͊g̀ͮ̇͌o͍ͧ t̜͕̱͐̑̏e̢͚̠̜ͩͯx̖ͦ͜͟͝ṱ̤̺ͩ͆̌ command

### PR DESCRIPTION
**Pull request by adam-zethraeus**
## Description

Converts text to z͚̳ͤ̓͐a̳̺ͥl̞̪͗͊g̀ͮ̇͌o͍ͧ t̜͕̱͐̑̏e̢͚̠̜ͩͯx̖ͦ͜͟͝ṱ̤̺ͩ͆̌

## Type of change

- [x]  New script command

## Screenshot
CleanShot 2021-04-11 at 17 05 20@2x

## Dependencies / Requirements

None

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by grzegorzkrukowski** [comment]
Thanks for your contribution.

The script works fine, but as it's silent at the beginning is hard to figure out if everything worked fine or if there were any errors during execution. As it's `silent` scripts - it's making Raycast disappear and showing nothing on success.

How about adding some confirmation message at the very end, like `R̴̨̝̍a̷̙̿͝ỷ̵̭c̸̠͂ä̶̝̝́s̶̺̽ţ̵̪̓́ copied to clipboard` ?

File: commands/conversions/zalgo-text.swift [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    return String(String.UnicodeScalarView(output))
+}
+
+NSPasteboard.general.clearContents()
+let text = CommandLine.arguments[1]
+let intensityString = CommandLine.arguments[2]
+let intensity = Int(intensityString) ?? 5
+let zalgoText = zalgo(text, intensity: intensity)
+NSPasteboard.general.setString(zalgoText, forType: .string)
+print("Clipboard: \(zalgoText)")
```
Comment (grzegorzkrukowski) [inline commands/conversions/zalgo-text.swift:49]:
```suggestion
print("\(zalgoText) copied to clipboard")
```

**Comment by adam-zethraeus** [comment]
thanks!

**Comment by grzegorzkrukowski** [comment]
I aligned the message to be similar as in other scripts - thanks for your contribution again.

**Pull request merged**